experiences/homepage/homepage.test.js

```diff
@@ -0,0 +1,7 @@
+const { expect } = require('chai');
+
+describe("homepage experience", function () {
+	it("test1", async function () {
+		expect(true).equal(true);
+	});
+});
```

Comment:
Tests are supposed to live here but will be consumed in https://github.com/Marfeel/flowcards as a package and get executed there.

In my mind, dependencies should be living here (`chai`, `wdio` ...) but at the same time there is no compilation process or an intention to run this code here.

What should be the approach here? @marcmrf @davidmohringmrf